.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+AllCops:
+  DisabledByDefault: true
```

Comment:
Did this sneak in? I wouldn't have expected rubocopping to have been affected by the changes.

Reply:
Nah I added this because RuboCop is setup to lint all my projects but there is no config in this project so it was yelling at me about every file I opened, haha. I can remove it but I was thinking maybe it would be a nudge to actually setup RuboCop for the project?